src/site/content/en/fast/browser-level-image-lazy-loading/index.md

```diff
@@ -35,9 +36,9 @@ without the need to write custom lazy-loading code or use a separate JavaScript
 
 ## Browser compatibility
 
-`<img loading=lazy>` is supported by most popular Chromium-powered browsers (Chrome, Edge, Opera)
-and [Firefox](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/75#HTML).
-The implementation for WebKit (Safari) [is in progress](https://bugs.webkit.org/show_bug.cgi?id=200764).
+`<img loading=lazy>` is supported by most popular Chromium-powered browsers (Chrome, Edge, Opera),
+[Firefox](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/75#HTML) and
```

Comment:
```suggestion
[Firefox](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/75#HTML), and
```

Oxford comma! (ooh these are fighting words)

Reply:
Not at all - I'm normally an Oxford comma enthusiast! But somehow this slipped through!

Thanks for the review and catching all those things.

Reply:
See, this is why we’re friends